lib/hooks/responses/index.js

```diff
@@ -212,9 +212,9 @@ function _mixin_jsonx(req, res) {
     sails.log.debug('***********************************************************************************');
 
     // Send conventional status message if no data was provided
-    // (see http://expressjs.com/api.html#res.send)
+    // (see http://expressjs.com/en/api.html#res.sendStatus)
     if (_.isUndefined(data)) {
-      return res.status(res.statusCode).send();
+      return res.sendStatus(res.statusCode);
```

Comment:
@dougwilson thanks!

@luislobo I think Scott actually ended up making the change you just made in the other default responses, so it totally makes sense-- but since jsonx is only still included for backwards compat., I think Doug's right and we want to just leave this bit as-is for compatibility's sake